tests/nightly/models/test_model_nightly.py

```diff
@@ -0,0 +1,130 @@
+"""Test Models on all MVTec Categories."""
```

Comment:
would `test_model.py` be enough since the parent directory already specifies the type of the test?